src/main/java/com/example/SteDziPanki/repository/CityRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.SteDziPanki.repository;
+
+import com.example.SteDziPanki.model.City;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface CityRepository extends JpaRepository<City, Long> {
```

Comment:
Jeżeli idkiem jest int (błąd!) to tutaj też powienien być int. 

Ogólnie Long jest ok, ale trzeba pozmieniać idki na longi